### ISSUE TITLE
Donwloading as Compressed Issue for Larger Files

### ISSUE THREAD

**Issue by mirzagee** [HuiiBuh/InstagramDownloader#356]
# Important: Issues without the following information will be closed

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to URL 'https://www.instagram.com/world_record_egg/'
3. See error

**Expected behaviour**

**Screenshots**

**Device (please complete the following information):**
- Resolution: [e.g 2K]
- Browser [e.g. chrome, firefox]
- Extension Version [e.g. 4.0.1]

**Additional context**

**Comment by mirzagee** [comment]
Expected behaviour
when the videos and pictures are more than 1000  it should download as zip but it is not happened 

Screenshots
it is quiting without throwing any error messages , only msg appears download completed 

Device (please complete the following information):

Resolution: [e.g 4K]
Browser [chrome]
Extension Version [4.8.5]
Additional context
If the files are downloading somwhere but not getting zipped to download 
which is continuously tried several times with posts more than 1000 in total

**Comment by crossfire27** [comment]
yes there shoud be option to download from date to date or from part to part.
